Things.md

```diff
@@ -0,0 +1,22 @@
+This is a general guide about Things for all FullScreenShenanigans projects. This guide will show you how in-game Things are manipulated and explain how the GroupHoldr system is used.  
```

Comment:
No need to mention the GroupHoldr system here, the average reader doesnt know what a group is.